src/app/zap-templates/zcl/data-model/chip/power-source-cluster.xml

```diff
@@ -24,44 +24,40 @@ limitations under the License.
     <client init="false" tick="false">true</client>
     <server init="false" tick="false">true</server>
     <description>This cluster is used to describe the configuration and capabilities of a physical power source that provides power to the Node.</description>
-    <!-- TODO: Add feature map once it is supported 
-    <tag name="WIRED" description="A wired power source"/>
```

Comment:
I would suggest to add now or later the definition of the FeatureMap as a bitmap8

Reply:
Yes, it is already defined below :)